babel.config.cjs

```diff
@@ -1,27 +1,34 @@
 const {getBabelConfig} = require('ocular-dev-tools/configuration');
 
+// eslint-disable-next-line no-process-env
+const pkg = require(process.env.npm_package_json);
+
 /**
  * NOTE: To debug our babel plugins, reference the local modules using
  * `./dev-modules/<plugin>/index.js`, instead of the npm package names.
  */
+
+const plugins = [
+  // inject __VERSION__ from package.json
+  // 'version-inline',
+  // [
+  //   // './dev-modules/babel-plugin-remove-glsl-comments/index.js'
+  //   'babel-plugin-remove-glsl-comments',
+  //   {patterns: ['**/shadertools/src/modules/**/*.js']}
+  // ]
+];
+
+// Lerna builds `@luma.gl/constants` before building any packages that depend
+// on it. This plugin uses LOCAL `@luma.gl/constants`, and will fail without
```

Comment:
Would it be better to just a pinned dependency on luma.gl/constants so we don't depend on the local copy?

This utility could just as well have been in its own repo, it was just a convenience to have it here, but since it seems to get involved in the overall build process that seems to create unnecessary problems.